windows-installer/build-installer.sh

```diff
@@ -102,15 +101,9 @@ tar -C tmp/jre/eclipse/jre -xaf packages/jre-*-windows-x64.tar.gz --strip-compon
 # call eclipse.exe instead of eclipsec.exe and that seemed to solve the
 # problem, but it seems I was mistaken.
 echo "Installing ARC plugins into Eclipse..."
-cmd /c tmp\\eclipse\\eclipse\\eclipsec.exe \
-    -application org.eclipse.equinox.p2.director \
-    -noSplash \
-    -repository ${ECLIPSE_REPO},jar:file:$(cygpath -w -a $IDE_PLUGINS_ZIP)\!/ \
-    -installIU ${ECLIPSE_PREREQ},com.arc.cdt.feature.feature.group
-# Eclipse will create a bunch of repos with local paths, that will not work for
-# end-users, hence those repos must be manually removed.
-sed -i -e "/$(echo "$(cygpath -w -a tmp)" | tr \\ _ | sed 's/[A-Z]://')/ d" \
-    tmp/eclipse/eclipse/p2/org.eclipse.equinox.p2.engine/profileRegistry/epp.package.cpp.profile/.data/.settings/org.eclipse.equinox.p2.*
+unzip ${IDE_PLUGINS_ZIP} -d tmp/eclipse/eclipse/dropins
```

Comment:
Yuu don't need to surround IDE_PLUGINS_ZIP with curly braces here, because it is already surrounded by whitespaces, thus it is clear what is the name of variable.